.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -18,10 +18,22 @@ If so, please describe the problem
 
 Include any specific requirements you need
 
+**Notify relevant community channels**
+
+Consider raising this topic on [Slack] in the #development channel or during a
+[community meeting] for awareness. List any relevant SIGs below from the
+[teams] list in the following form so that members of those teams are notified:
```

Comment:
Not obvious to me what "List" means here, but is this right? 

```suggestion
[community meeting] for awareness. Notify the members of any relevant SIGs from the
[teams] list by mentioning them in the issue in the following format:
```

Reply:
Applied along with Bill's suggestion, thanks.